packages/smooth_ui_library/lib/widgets/smooth_card.dart

```diff
@@ -12,8 +12,8 @@ class SmoothCard extends StatelessWidget {
   });
 
   final Widget child;
-  final Color color;
-  final Widget header;
+  final Color? color;
+  final Widget? header;
   final EdgeInsets padding;
   final EdgeInsets insets;
```

Comment:
A bit lower in the code, there's a `Column` with one child as `header ?? Container()`.
I guess `if (header != null) header` would be cleaner.